src/pages/Contact.jsx

```diff
@@ -23,15 +23,15 @@ const Contact = () => {
           <div className="mt-[60px]">
             <div className="flex gap-[20px]">
               <div className="flex flex-col text-grey">
-                <label htmlFor="Email" className="text-grey font-medium">Name</label>
```

Comment:
For the input fields you could make it a reusable component so as not to repeat classNames